File: C/1045.c
#include <stdio.h>
 
int main() {
 
    float a,b,c,temp;
    scanf("%f%f%f", &a,&b,&c);
    
    if(a < b){
        temp = a;
        a = b;
        b = temp;
    }if(b < c){
        temp = b;
        b = c;
        c = temp;
    }if(a < b){
        temp = a;
        a = b;
        b = temp;
    }
    
    if(a >= (b + c)){
        printf("NAO FORMA TRIANGULO\n");
    }else if(a*a == b*b + c*c){
        printf("TRIANGULO RETANGULO\n");
    }else if(a*a > b*b + c*c){
        printf("TRIANGULO OBTUSANGULO\n");
    }else if(a*a < b*b + c*c){
        printf("TRIANGULO ACUTANGULO\n");
    }if(a == b && b == c){
        printf("TRIANGULO EQUILATERO\n");
    }else if(a == b | a == c | b == c){
        printf("TRIANGULO ISOSCELES\n");
    }
}

File: Python/1045.py
a,b,c=list(map(float,input().split()))
if(a < b):
    temp = a
    a = b
    b = temp
if(b < c):
    temp = b
    b = c
    c = temp
if(a < b):
    temp = a
    a = b
    b = temp

if (a >= (b+c)):
    print('NAO FORMA TRIANGULO')
elif a**2 == b**2 + c**2:
    print('TRIANGULO RETANGULO')
elif a**2 > b**2 + c**2:
    print('TRIANGULO OBTUSANGULO')
elif a**2 < b**2 + c**2:
    print('TRIANGULO ACUTANGULO')
if a == b == c:
    print('TRIANGULO EQUILATERO')
elif a == b != c or a == c != b or b == c != a:
    print('TRIANGULO ISOSCELES')